kythe/go/serving/xrefs/xrefs.go

```diff
@@ -292,6 +293,11 @@ func (t *Table) Decorations(ctx context.Context, req *xpb.DecorationsRequest) (*
 		bindings := stringset.New()
 
 		for _, d := range decor.Decoration {
+			// Filter decorations by requested build configs.
+			if len(buildConfigs) != 0 && !buildConfigs.Contains(d.Anchor.BuildConfiguration) {
+				continue
+			}
+
```

Comment:
This filter decorations. Isn't it possible that asking for xrefs for a ticket might also need filtering? (i.e. if the xref is in some other build config)?

Reply:
Filtering for xrefs will be done in a separate change.